src/main/java/com/exadel/discount/service/SortPageMaker.java

```diff
@@ -0,0 +1,15 @@
+package com.exadel.discount.service;
+
+import org.springframework.data.domain.PageRequest;
+import org.springframework.data.domain.Pageable;
+import org.springframework.data.domain.Sort;
+
+public class SortPageMaker {
+    public static Pageable makePageable(int pageNumber, int pageSize, String sortDirection, String sortField) {
+        Sort sort = Sort.unsorted();
+        if (("ASC").equals(sortDirection.toUpperCase()) || ("DESC").equals(sortDirection.toUpperCase())) {
```

Comment:
Brackets are not necessary in ("ASC"), and you still can use equalsIgnoreCase

Reply:
Done